pkg/sql/isql/options.go

```diff
@@ -50,6 +50,12 @@ func WithSessionData(sd *sessiondata.SessionData) Option {
 	return (*sessionDataOption)(sd)
 }
 
+// WithSkipDescriptorCache forces the internal executor to avoid all leasing
```

Comment:
Could you add a short not about when or why this option might be used and any warnings that users should consider if they do opt to use this option?